resources/sample-config.json

```diff
@@ -371,6 +371,13 @@
 			"props": {
 				"severity": "ERROR"
 			}
+		},
+		{
+			"type": "NullParameter",
+			"props": {
+				"severity": "ERROR",
+				"nullDefaultValueStyle": "questionmark"
```

Comment:
Is there a convention for the naming of the option values? Sometimes they're UPPERCASE, (`ENUM`) sometimes lowercase (`nl`).